azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_nuget.go

```diff
@@ -0,0 +1,194 @@
+package serviceendpoint
```

Comment:
There should be some build comments/commands at the top of this file.


```suggestion
//go:build (all || resource_serviceendpoint_nuget) && !exclude_serviceendpoints
// +build all resource_serviceendpoint_nuget
// +build !exclude_serviceendpoints

package serviceendpoint
```

Think that'd be correct

Reply:
Well apparently it doesn't go in this file :/